xds/src/main/proto/centraldogma/xds/k8s/v1/xds_kubernetes.proto

```diff
@@ -19,70 +19,89 @@ option java_multiple_files = true;
 option java_outer_classname = "XdsKubernetesProto";
 option java_package = "com.linecorp.centraldogma.xds.k8s.v1";
 
+import "envoy/config/core/v3/base.proto";
+import "envoy/config/endpoint/v3/endpoint.proto";
+
 import "google/api/annotations.proto";
 import "google/api/client.proto";
 import "google/api/field_behavior.proto";
 import "google/protobuf/empty.proto";
+import "google/protobuf/wrappers.proto";
 
 service XdsKubernetesService {
 
-  rpc CreateServiceEndpointWatcher(CreateServiceEndpointWatcherRequest) returns (ServiceEndpointWatcher) {
+  rpc CreateKubernetesEndpointAggregator(CreateKubernetesEndpointAggregatorRequest)
+      returns (KubernetesEndpointAggregator) {
     option (google.api.http) = {
-      post: "/api/v1/xds/{parent=groups/*}/k8s/watchers"
-      body: "watcher"
+      post: "/api/v1/xds/{parent=groups/*}/k8s/endpointAggregators"
```

Comment:
Used camel case for the collection identifiers:
https://github.com/aip-dev/google.aip.dev/blob/master/aip/general/0122.md#collection-identifiers